lib/bolt/inventory/group.rb

```diff
@@ -75,9 +69,9 @@ def initialize(data)
             @logger.warn(msg)
           end
 
-          next unless node.include?('alias')
-
-          aliases = node['alias']
+          # Add aliases before checking whether target is already in group, otherwise they will
```

Comment:
I dont think this is what we want. If the target is already in the group, all the config for that target should be ignored I think (including an alias entry).